docs/key-issues/gpus.md

```diff
@@ -30,52 +30,66 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
+[NVIDIA's Private PyPI Index](https://pypi.org/project/nvidia-pyindex/), which
+also includes rebuilds of TensorFlow and other packages.
 
 A single CUDA version supports a reasonable range of GPU architectures. New
 CUDA versions get released regularly, and - because they come with increased
 performance or new functionality - it may be necessary or desirable to build
 new wheels for that CUDA version. If only the supported CUDA version is
 different between two wheels, the wheel tags and filename will be identical.
 Hence it is not possible to upload more than one of those wheels under the same
-package name. To work around that, a project may either support only one CUDA
-version on PyPI, or create different packages. PyTorch and TensorFlow do the
-former, with TensorFlow supporting only a single CUDA version, and PyTorch
-providing more wheels for other CUDA versions and a CPU-only version in a
-separate wheelhouse (see
+package name.
+
+Historically, this required projects to produce packages specific CUDA minor
+versions. Projects would either support only one CUDA version on PyPI, or create
+different packages. PyTorch and TensorFlow do the former, with TensorFlow
+supporting only a single CUDA version, and PyTorch providing more wheels for
+other CUDA versions and a CPU-only version in a separate wheelhouse (see
 [pytorch.org/get-started](https://pytorch.org/get-started/locally/)).
 CuPy provides a number of packages: [`cupy`](https://pypi.org/project/cupy/),
 [`cupy-cuda102`](https://pypi.org/project/cupy-cuda102/),
 [`cupy-cuda110`](https://pypi.org/project/cupy-cuda1110/),
 [`cupy-cuda111`](https://pypi.org/project/cupy-cuda111/),
-[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/),
 [`cupy-rocm-4-3`](https://pypi.org/project/cupy-rocm-4-3/),
 [`cupy-rocm-5-0`](https://pypi.org/project/cupy-rocm-5-0/).
-Other projects do similar things - none of it works very well.
-
-GPU packages tend to result in very large wheels. This is true in particular
-for deep learning packages, because they link in
-[cuDNN](https://developer.nvidia.com/cudnn). For example, the most recent
-`manylinux2014` wheels for TensorFlow are 588 MB
+As of CUDA 11, CUDA promises [minor version
+compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility),
+which allows building packages compatibility across an entire CUDA major
+version. CuPy now leverages this to produce wheels like 
+[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/) and
+[`cupy-cuda12x`](https://pypi.org/project/cupy-cuda12x/) that work for any CUDA
+11.x or CUDA 12.x version, respectively, that a user has installed.
+
+GPU packages tend to result in very large wheels. This is in large part because
+compiled GPU libraries must support a number of architectures, leading to large
+binary sizes. These effects are compounded by the requirements imposed by the
+manylinux standard for Linux wheels, which results in many large libraries being
```

Comment:
I wonder if we should add a link to the "Depending on packages for which an ABI matters" section (on another page) since we're touching manylinux here. No strong opinion, though.

Reply:
I'm not opposed, but I'm not sure exactly how that would look here. Perhaps just a "see also [the notes on ABI compatibility](...)"?

Reply:
I think it's actually more this page where vendoring requirements are explained: https://pypackaging-native.github.io/key-issues/native-dependencies/. So I suggest adding:
```
(see [Native dependencies](native-dependencies) for details)
```